frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.styled.tsx

```diff
@@ -59,3 +60,11 @@ export const ContentSection = styled.div`
 export const DescriptionHeader = styled.h3`
   margin-bottom: 0.5rem;
 `;
+
+export const SpaceBetweenSwitch = styled(Switch)`
+  > div {
```

Comment:
We can use the [body class]([url](https://v6.mantine.dev/core/switch/?t=styles-api)) exposed by mantine, so instead this line would be

```css
  .emotion-Switch-body {
```

Reply:
Feels like a design system hack. I don't think we should do this. In the worst case just don't pass `label`.

Reply:
FWIW @mazameli did explicitly ask if we can support `space-between` on the component https://github.com/metabase/metabase/pull/34005#issuecomment-1736399716

Reply:
I think a little communication whoopsie happened with the original component, but it's no big deal. I talked to @mazameli and the intent is that this "stretch" to fill Switch should be part of our Switch component's option space. I'm with @ranquild that it feels like we're doing something messy if in order to get that to happen we have to write CSS at each point of use.

Happily I think we can use [a custom variant](https://v7.mantine.dev/styles/variants-sizes#adding-custom-variants) and just bake this behavior into Switch itself.

So I think to fix that piece, we do the following
- Add a custom `stretchy` variant to Swtich, and add this `flex` and `space-between` CSS to that variant in the style for the component.
- Document it 
- Use that here `<Switch variant="stretch" labelPosition="left" ... />`

Reply:
Can do. 👍  I love working with designers. ❤️